start-minecraftFinalSetup

```diff
@@ -96,6 +96,10 @@ if [ -n "$JVM_DD_OPTS" ]; then
       done
 fi
 
+if [ -z "$SERVER_IP" ]; then
```

Comment:
...yes, go ahead and remove this block since it’s too late in the container lifecycle to be used by healthcheck